includes/validation/class-amp-validation-manager.php

```diff
@@ -262,7 +262,7 @@ public static function post_supports_validation( $post ) {
 		}
 
 		// Story post type always supports validation.
-		if ( AMP_Story_Post_Type::POST_TYPE_SLUG === $post->post_type ) {
+		if ( AMP_Story_Legacy_Post_Type::POST_TYPE_SLUG === $post->post_type ) {
```

Comment:
Probably we'd still need AMP validation for the new Stories post as well?

Reply:
I think we should break this functionality onto issue / PR.

Reply:
Maybe #3838 is a better place to address this.

Reply:
This PR is just for creating a post type and moving some files around. I would like to keep it as small as possible. Porting over validation makes more sense in another ticket.